README.md

```diff
@@ -171,6 +172,7 @@ navPrev: PropTypes.node,
 navNext: PropTypes.node,
 onPrevMonthClick: PropTypes.func,
 onNextMonthClick: PropTypes.func,
+onClose: ({ startDate, endDate }) => this.setState({ startDate, endDate }) // PropTypes.func,
```

Comment:
I put the function to let the user know how to use the callback... I didn't what would be the better way to show it. Any ideias?